crates/bevy_render/src/color/mod.rs

```diff
@@ -487,6 +487,15 @@ impl Color {
             } => [hue, saturation, lightness, alpha],
         }
     }
+
+    pub fn as_linear_abgr_u32(self) -> u32 {
```

Comment:
Can this be a `const fn`?

Reply:
Only if `as_linear_rgba_f32` and friends, are also made `const fn`. Not my call to make.

It should probably have an inline annotation, though. Added that.